contracts/minters/vending-minter/src/contract.rs

```diff
@@ -690,9 +701,15 @@ fn _execute_mint(
         .add_attribute("sender", info.sender)
         .add_attribute("recipient", recipient_addr)
         .add_attribute("token_id", mintable_token_mapping.token_id.to_string())
-        .add_attribute("network_fee", network_fee)
-        .add_attribute("mint_price", mint_price.amount)
-        .add_attribute("seller_amount", seller_amount))
+        .add_attribute(
+            "network_fee",
+            coin(network_fee.u128(), mint_price.clone().denom).to_string(),
+        )
```

Comment:
I don't think indexers use this but this will probably break it since going from number to amount+denom

Reply:
Right, indexer will have to be updated to parse full coins for this new version and beyond.